tuna/worker_interface.py

```diff
@@ -142,53 +159,54 @@ def set_logger(self, logger_name):
     lgr.setLevel(log_level.upper() if log_level else logging.DEBUG)
     self.logger = lgr
 
-  def set_db_tables(self):
+  def set_db_tables(self) -> None:
     """Initialize tables"""
-    self.dbt = DBTablesInterface(session_id=self.session_id)
+    self.dbt: DBTablesInterface = DBTablesInterface(session_id=self.session_id)
 
-  def reset_machine(self):
+  def reset_machine(self) -> None:
     """Function to reset machhine"""
     self.machine.restart_server()
     self.last_reset = datetime.now()
 
   def compose_work_objs(self, session: DbSession,
-                        conds: List[str]) -> List[Tuple[SimpleDict, ...]]:
+                        cmds: List[str]) -> List[SimpleDict]:
     """Query a job list for update"""
-    cond_str = ' AND '.join(conds)
-    if cond_str:
-      cond_str = f"WHERE {cond_str}"
-    cond_str += f" ORDER BY retries ASC LIMIT {self.claim_num} FOR UPDATE"
+    cmd_str = ' AND '.join(cmds)
+    if cmd_str:
+      cmd_str = f"WHERE {cmd_str}"
+    cmd_str += f" ORDER BY retries ASC LIMIT {self.claim_num} FOR UPDATE"
     #try once without waiting for lock
-    no_lock = cond_str + " SKIP LOCKED"
+    no_lock = cmd_str + " SKIP LOCKED"
     entries = gen_select_objs(session, self.job_attr,
                               self.dbt.job_table.__tablename__, no_lock)
     if not entries:
       entries = gen_select_objs(session, self.job_attr,
-                                self.dbt.job_table.__tablename__, cond_str)
+                                self.dbt.job_table.__tablename__, cmd_str)
 
     return [(job,) for job in entries]
 
   def get_job_objs(self, session: DbSession,
-                   find_state: str) -> List[Tuple[SimpleDict, ...]]:
+                   find_state: str) -> List[SimpleDict]:
```

Comment:
Return type is incorrect, see return of compose_work_objects. This will depend on implementation, but at this level generically: List[Tuple[SimpleDict, ...]]